easybuild/easyconfigs/d/DeltaLake/DeltaLake-0.15.1-gfbf-2023a.eb

```
@@ -0,0 +1,873 @@
+    ('thiserror', '1.0.56'),
+    ('thiserror-impl', '1.0.56'),
+    ('thrift', '0.17.0'),
+    ('time', '0.3.31'),
+    ('time-core', '0.1.2'),
+    ('time-macros', '0.2.16'),
+    ('tiny-keccak', '2.0.2'),
+    ('tinytemplate', '1.2.1'),
+    ('tinyvec', '1.6.0'),
+    ('tinyvec_macros', '0.1.1'),
+    ('tokio', '1.35.1'),
+    ('tokio-macros', '2.2.0'),
+    ('tokio-native-tls', '0.3.1'),
+    ('tokio-rustls', '0.22.0'),
+    ('tokio-rustls', '0.24.1'),
+    ('tokio-util', '0.7.10'),
+    ('tower-service', '0.3.2'),
+    ('tracing', '0.1.40'),
+    ('tracing-attributes', '0.1.27'),
+    ('tracing-core', '0.1.32'),
+    ('try-lock', '0.2.5'),
+    ('twox-hash', '1.6.3'),
+    ('typenum', '1.17.0'),
+    ('unicode-bidi', '0.3.14'),
+    ('unicode-ident', '1.0.12'),
+    ('unicode-normalization', '0.1.22'),
+    ('unicode-segmentation', '1.10.1'),
+    ('unicode-width', '0.1.11'),
+    ('unindent', '0.2.3'),
+    ('untrusted', '0.7.1'),
+    ('untrusted', '0.9.0'),
+    ('url', '2.5.0'),
+    ('urlencoding', '2.1.3'),
+    ('utime', '0.3.1'),
+    ('uuid', '1.6.1'),
+    ('vcpkg', '0.2.15'),
+    ('version_check', '0.9.4'),
+    ('vsimd', '0.8.0'),
+    ('walkdir', '2.4.0'),
+    ('want', '0.3.1'),
+    ('wasi', '0.11.0+wasi-snapshot-preview1'),
+    ('wasm-bindgen', '0.2.89'),
+    ('wasm-bindgen-backend', '0.2.89'),
+    ('wasm-bindgen-futures', '0.4.39'),
+    ('wasm-bindgen-macro', '0.2.89'),
+    ('wasm-bindgen-macro-support', '0.2.89'),
+    ('wasm-bindgen-shared', '0.2.89'),
+    ('wasm-streams', '0.3.0'),
+    ('web-sys', '0.3.66'),
+    ('webpki', '0.21.4'),
+    ('webpki-roots', '0.25.3'),
+    ('winapi', '0.3.9'),
+    ('winapi-i686-pc-windows-gnu', '0.4.0'),
+    ('winapi-util', '0.1.6'),
+    ('winapi-x86_64-pc-windows-gnu', '0.4.0'),
+    ('windows-core', '0.52.0'),
+    ('windows-sys', '0.48.0'),
+    ('windows-sys', '0.52.0'),
+    ('windows-targets', '0.48.5'),
+    ('windows-targets', '0.52.0'),
+    ('windows_aarch64_gnullvm', '0.48.5'),
+    ('windows_aarch64_gnullvm', '0.52.0'),
+    ('windows_aarch64_msvc', '0.48.5'),
+    ('windows_aarch64_msvc', '0.52.0'),
+    ('windows_i686_gnu', '0.48.5'),
+    ('windows_i686_gnu', '0.52.0'),
+    ('windows_i686_msvc', '0.48.5'),
+    ('windows_i686_msvc', '0.52.0'),
+    ('windows_x86_64_gnu', '0.48.5'),
+    ('windows_x86_64_gnu', '0.52.0'),
+    ('windows_x86_64_gnullvm', '0.48.5'),
+    ('windows_x86_64_gnullvm', '0.52.0'),
+    ('windows_x86_64_msvc', '0.48.5'),
+    ('windows_x86_64_msvc', '0.52.0'),
+    ('winreg', '0.50.0'),
+    ('xml-rs', '0.8.19'),
+    ('xmlparser', '0.13.6'),
+    ('xz2', '0.1.7'),
+    ('yansi', '0.5.1'),
+    ('z85', '3.0.5'),
+    ('zerocopy', '0.7.32'),
+    ('zerocopy-derive', '0.7.32'),
+    ('zeroize', '1.7.0'),
+    ('zstd', '0.13.0'),
+    ('zstd-safe', '7.0.0'),
+    ('zstd-sys', '2.0.9+zstd.1.5.5'),
+]
+
+builddependencies = [
+    ('maturin', '1.4.0', '-Rust-1.75.0'),
+    ('Perl', '5.36.1'),  # needed by crate openssl-sys 
```

remove trailing whitespace that is tripping up code style check:
```suggestion
    ('Perl', '5.36.1'),  # needed by crate openssl-sys
```